pkg/executor/executortype/newdeploy/newdeploymgr.go

```diff
@@ -755,6 +755,9 @@ func (deploy *NewDeploy) getDeployLabels(fnMeta metav1.ObjectMeta, envMeta metav
 
 func (deploy *NewDeploy) getDeployAnnotations(fnMeta metav1.ObjectMeta, envMeta metav1.ObjectMeta) map[string]string {
 	deployAnnotations := maps.CopyStringMap(envMeta.Annotations)
+	for k, v := range fnMeta.Annotations {
```

Comment:
Can we please do a similar change in the above getDeployLabels also? We can copy function labels as well.